app/controllers/ncr/work_orders_controller.rb

```diff
@@ -17,30 +16,28 @@ def edit
       if self.proposal.approved?
         flash[:warning] = "You are about to modify a fully approved request. Changes will be logged and sent to approvers but this request will not require re-approval."
       end
-      first_approver = self.proposal.approvers.first
-      @approver_email = first_approver.try(:email_address)
 
       super
     end
 
     def update
-      @approver_email = params[:approver_email]
       @model_instance.modifier = current_user
 
       super
 
       if @model_changing && !@model_instance.emergency # skip approvals if emergency
-        @model_instance.setup_approvals_and_observers(@approver_email)
+        @model_instance.setup_approvals_and_observers
         @model_instance.email_approvers
       end
     end
 
+    protected
+
     def attribute_changes?
-      super || @model_instance.approver_changed?(@approver_email)
+      # TODO remove need for passing in the approving_official_email
```

Comment:
let's put this in Tracker instead! (and add more details on why etc while we're at it!)